### DISCUSSION TITLE
Apple Pencil / Scribble Support

### DISCUSSION THREAD

**Discussion by daverad** [Flutter-Bounty-Hunters/super_editor#1910] [Ideas]
We've gotten a few more requests recently for Apple Pencil support. Is this something that could be possible with SuperEditor?

**Comment by Jethro87** [comment]
Hey @daverad! We're interested in Scribble too, and are willing to co-fund it. Will your team be able to sponsor some hours to add this?

**Reply by daverad** [reply]
hey hey! its not very high prio for us yet, but happy to sponsor a couple hours if it helps move things along!
